source/ocs/ocs-connection.ts

```diff
@@ -39,7 +39,18 @@ export class OcsConnection {
       callback(error, null);
       return;
     }
-    const jsonBody = JSON.parse(body || '{}');
+
+    let jsonBody;
+
+    try {
+      jsonBody = JSON.parse(body || '{}');
+    } catch {
+      callback({
+        code: 500,
+        message: 'The response body is not a valid JSON.'
```

Comment:
`Unable to parse the response body as valid JSON`

`a valid JSON` isn't correct and variations thereof are also sort of weird.